src/safe-global/index.ts

```diff
@@ -7,12 +7,46 @@ import {
 } from '@safe-global/safe-gateway-typescript-sdk';
 import fetch from 'cross-fetch';
 
+import { Maybe } from '../types';
 import { errorMessageForError } from '../utilities';
+import { SafeGlobalAccountDetails } from './types';
 
 class SafeGlobalClient {
+  async getAccountDetails(
+    network: string,
+    safeAddress: string
+  ): Promise<Maybe<SafeGlobalAccountDetails>> {
+    try {
+      // https://safe-client.safe.global/v1/chains/1/safes/0x2A83Aaf231644Fa328aE25394b0bEB17eBd12150
```

Comment:
Delete this if not needed

Reply:
:), I left this comment to remember what the original URL was. I think it would be helpful after a while.

Reply:
Okay, that's fine then just add a line above it explaining why it's there